decode.go

```diff
@@ -48,31 +48,43 @@ func parseLocalDate(b []byte) (LocalDate, error) {
 	if err != nil {
 		return date, err
 	}
+
 	date.Month = time.Month(v)
 
 	date.Day, err = parseDecimalDigits(b[8:10])
+	if err != nil {
+		return date, err
+	}
 
 	return date, nil
 }
 
+var ErrNotDigit = errors.New("not a digit")
```

Comment:
I'd like to avoid exporting errors at the moment, as I haven't fleshed out what the API for handling errors should be.

Reply:
This is my mistake, I should not have capitalized them right away, I will lowercase all the errors.